bevy_matchbox/Cargo.toml

```
@@ -21,10 +21,10 @@ readme = "../README.md"
 
 [features]
 ggrs = ["matchbox_socket/ggrs"]
-signaling = ["dep:matchbox_signaling", "dep:async-compat"]
+signaling = ["dep:matchbox_signaling", "dep:async-compat", "bevy/multi-threaded"]
```

```suggestion
signaling = [
  "dep:matchbox_signaling",
   "dep:async-compat",
   "bevy/multi-threaded"
]
```

I *think* this is what taplo wants, though I haven't checked. Not sure why we don't get more specific errors in CI...